diqu/utils/jira_variable_config.py

```diff
@@ -0,0 +1,4 @@
+JIRA_ISSUE_TYPE = "Bug"
+JIRA_OPEN_TICKETS_FILTER_BY_SUMMARY = "dq-tools"
+JIRA_TICKET_DEPRECATED_WINDOW_IN_DAYS = 3
+JIRA_TICKET_UPDATE_WINDOW_IN_DAYS = 14
```

Comment:
Let's remove this file for now, and use all as the environment variables.
In the future, we will use config.yml file to configure these values.